units/src/parse.rs

```diff
@@ -78,6 +78,10 @@ mod sealed {
 ///
 /// If the caller owns `String` or `Box<str>` which is not used later it's better to pass it as
 /// owned since it avoids allocation in error case.
+///
+/// # Panics
+///
+/// Panics if the type is larger than 128 bits.
```

Comment:
In f0de79b1cb5ad53eaf78986ee61f4b878bec4c4d:

Strictly speaking the panic is only if the type is larger than *255* bits. This doc is probably reasonable.

Though we could just change the error type to store a string representation or the `TypeId` or something of `T` rather than using the bit count.

Reply:
I think we should also change this from 128 to 255.

Reply:
Done